IPython/core/shellapp.py

```diff
@@ -139,7 +139,18 @@ class InteractiveShellApp(Configurable):
     extra_extension = Unicode('', config=True,
         help="dotted module name of an IPython extension to load."
     )
-    
+
+    def _extra_extension_changed(self, name, old, new):
```

Comment:
I think I picked up an extraneous diff in my rebase...not sure where this came from.